acceptance/analyzer_test.go

```diff
@@ -206,11 +205,7 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 			it("recursively chowns the directory", func() {
 				h.SkipIf(t, runtime.GOOS == "windows", "Not relevant on Windows")
 
-				imgArg := ""
-				// Platform >= 0.7 does not allow an argument, < 7 requires an argument
-				if api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0 {
-					imgArg = "some-image"
-				}
+				imgArg := "some-image"
```

Comment:
Could we just inline this?

Reply:
Could we change the occurrences of `"-previous-image", "some-image"` in this file to just `"some-image"`, so that it's more realistic?

Also, the only tests left in this file that use `-previous-image` are ones that expect an error. Is there a way to show that we correctly process this flag when the previous image and the destination image aren't the same?

Reply:
Fixed in 942ee2c

Reply:
Ahh the tests are easier to read now! I noticed that we're still not using `-previous-image` anywhere - could we just add a new context like `when("called with previous image"` that uses `some-image` as the previous image and tries to write to `some-new-image` (or something). That way we can be sure that we're correctly parsing this argument.

Reply:
See 0e2047a!